backend/src/main/java/ch/ase21/backend/service/SaleService.java

```diff
@@ -248,6 +248,8 @@ private SaleService() {/* void */}
 
     if(neighbourhoodScore == null){
       neighbourhoodScore = 0.5;
```

Comment:
Shouldn't this be 5.0?

Reply:
nope, needs to be between 0 and 1 for the overall calculation